src/pages/EditarPerfil/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Header from '../../components/Header';
+//import Footer from './components/Footer';
+import Button from '../../components/Button';
+import Input from '../../components/Input';
+import UserImage from '../../components/UserImage';
+
+import "./styles.css";
+import profile_user_image from "../../assets/images/cintialorenzzo.jpeg";
+import camera_icon from "../../assets/awesome-camera.svg";
+
+const EditarPerfil = () => (
+    <div>
+        <Header />
+        <div className="container-editar-perfil">
+            <div className="user-image">
+                <UserImage source={profile_user_image} />
+                <div className="container-alterar-foto" href="#">
+                    <img src={camera_icon} />
+                    Alterar sua foto de perfil
```

Comment:
É importante colocar tag em todos os textos!
```suggestion
                   <span> Alterar sua foto de perfil </span>
```